apps/token/src/routes/governance/components/proposal-change-table/proposal-change-table.tsx

```diff
@@ -19,26 +16,26 @@ export const ProposalChangeTable = ({ proposal }: ProposalChangeTableProps) => {
   const terms = proposal.terms;
 
   return (
-    <KeyValueTable muted={true} data-testid="proposal-change-table">
-      <KeyValueTableRow>
+    <KeyValueTable data-testid="proposal-change-table">
+      <KeyValueTableRow muted={true}>
```

Comment:
muted and numerical tailwind styling has been added directly on `tr`,  `td` and `th` tags. This is why these props have been moved to KeyValueTableRows. I am looking into passing these props to the children.